client/gutenberg/extensions/contact-form/editor.js

```diff
@@ -24,12 +24,12 @@ import registerJetpackBlock from 'gutenberg/extensions/presets/jetpack/utils/reg
  * Block Registrations:
  */
 registerJetpackBlock( 'contact-form', {
-	title: __( 'Contact Form' ),
+	title: __( 'Form' ),
 	description: __( 'A simple way to get feedback from folks visiting your site.' ),
 	icon: renderMaterialIcon(
 		<path d="M13 7.5h5v2h-5zm0 7h5v2h-5zM19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 16H5V5h14v14zM11 6H6v5h5V6zm-1 4H7V7h3v3zm1 3H6v5h5v-5zm-1 4H7v-3h3v3z" />
 	),
-	keywords: [ __( 'email' ), __( 'feedback' ), 'email' ],
+	keywords: [ __( 'email' ), __( 'feedback' ), __( 'contact' ) ],
```

Comment:
does it search the Title or we need to add `form` here too? in any case I would also add `form` untranslated to the keywords

Reply:
You don't have to add `form` here since that would be searched by default

Reply:
So do I need to change something here? I think Form will already be translated.